commands/command_set.gd

```diff
@@ -12,14 +12,12 @@ const PlusOperables = [
 	TYPE_ARRAY, 29, 30, 31, 32, 33, 34, 35, 36, 37
 ]
 
-## If [code]true[/code], the [param value] will be [b]added[/b] instead of [b]set[/b].[br]
-## To [b]subtract[/b], use a negative value like [param value] of -1
-var add_value:bool = false:
```

Comment:
This one was deprecated too, better to just mark as deprecated instead of removing

Reply:
no needed